admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+cluster and defines secrets and templates to implement aggregated logging.
+
+. Create a new project. Once implemented in a single project, the EFK stack will apply to every project within your OpenShift cluster:
++
+----
+$ oc new-project <project_name>
+----
+
+. Create a link:../dev_guide/secrets.html[secret] for the deployer. The secret can be configured for your
+instance, but specific for the deployer.
++
+A set of files can be supplied in the secret:
++
+[cols="2",options="header"]
+|===
+|File Name
+|Description
+
+|*_kibana.crt_*
+|A browser-facing certificate for the Kibana server.
+
+|*_kibana.key_*
+|A key to be used with the Kibana certificate.
+
+|*_kibana-ops.crt_*
+|A browser-facing certificate for the Ops Kibana server.
+
+|*_kibana-ops.key_*
+|A key to be used with the Ops Kibana certificate.
+
+|*_server-tls.json_*
+|JSON TLS options to override the Kibana server defaults. Refer to NodeJS docs for available options and the default options for an example.
+
+|*_ca.crt_*
+|A certificate for a CA that will be used to sign all certificates generated by the deployer.
+
+|*_ca.key_*
+|A matching CA key.
+|===
++
+You can supply the files when creating a new secret:
++
+----
+$ oc secrets new logging-deployer \
+   kibana.crt=/path/to/cert kibana.key=/path/to/key
+----
+
+. Create the deployer link:../admin_guide/service_accounts.html[service account]:
++
+====
+----
+oc create -f - <<API
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: logging-deployer
+secrets:
+- name: logging-deployer
+API
+----
+====
++
+Then, ensure the logging-deployer can edit the service:
++
+====
+----
+$ oc policy add-role-to-user edit \
+          system:serviceaccount:logging:logging-deployer
+----
+====
+
+== Deploying the EFK Stack
+
+Deploy the entire EFK stack link:../dev_guide/templates.html[using a template],
+ensuring all future project configurations will feature the EFK stack. The
+default repository and branch of the template are set to origin/master. Change
+these settings to suit your own instance if desired. The template is found at
+the following GitHub repository:
+
+https://github.com/openshift/origin-aggregated-logging/blob/master/hack/templates/dev-builds.yaml
+
+[IMPORTANT]
+====
+The openshift-auth-proxy depends on a "node" base image, which is intended to be
+the DockerHub nodejs base image. Defining standard templates also depends on a
+nodejs image called "node", which will be used instead of the intended image,
+causing the build to fail. To resolve this, delete the older node image:
+
+----
+$ oc delete is/node -n openshift
+----
+====
+
+After defining any builds, they should restart. If not, you can restart each
+component:
+
+----
+$ oc start-build <component>
+----
+
+Once the builds have successfully completed, the image streams will be populated
+and ready for deployment. However, you must first specify the `*IDEX_PREFIX*`
+parameter with the registry location.
+
```

Comment:
@sosiouxme Do you mean just the above paragraph? And do you mean with the info that you and @adellape were talking about above?